conf/config-default.yaml

```diff
@@ -274,6 +274,7 @@ etcd:
   timeout: 30                     # 30 seconds
   #resync_delay: 5                # when sync failed and a rest is needed, resync after the configured seconds plus 50% random jitter
   #health_check_timeout: 10       # etcd retry the unhealthy nodes after the configured seconds
+  health_check_retry: 2           # etcd retry time, defalut 2 
```

Comment:
We should emphasize that the retry time only affects the health check in the comment.

Reply:
thx, i will specify it